tests/io/datasets/test_k2is.py

```diff
@@ -29,6 +32,16 @@ def default_k2is():
     return ds
 
 
+@pytest.fixture(scope='module')
+def default_k2is_raw():
+    return np.memmap(
```

Comment:
Could we make the raw reference data optional, too? I do have the K2 reference data on my laptop, for example, but I don't think I have space to also keep raw references around - what do you think?

Reply:
Done! :-)